xwiki-pro-macros-ui/src/main/resources/XWiki/Macros/Panel.xml

```diff
@@ -540,7 +564,16 @@ This macro supports all Atlassian Confluence parameters as of version 7.9.</cont
         $!xcontext.macro.content
       #end
     )))
-  )))
+    #if( $panelFooter != '')
+      (% class="macro-panel-footer" style="$panelFooterStyle"%)(((
+        ## Since 11.5 the content of the macro can be edited in WYSIWYG editor.
+        #if ($services.rendering.getMacroDescriptor($services.rendering.resolveMacroId('wikimacroparameter')))
```

Comment:
this if is not needed since the parent of our app is 13.10 and the wikimacroparameter exists since 11.5 https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-wikimacro/xwiki-platform-rendering-wikimacro-store/src/main/java/org/xwiki/rendering/wikimacro/internal/WikiMacroParameterMacro.java